frameworks/Python/aiohttp/app/main.py

```diff
@@ -1,7 +1,7 @@
 import os
 import multiprocessing
 
-import asyncpg
+import asyncpg.connection
```

Comment:
I don't think this line should have changed.
```suggestion
import asyncpg
```